elasticdl/docker/Dockerfile

```diff
@@ -6,7 +6,7 @@ FROM ${BASE_IMAGE}
 COPY elasticdl/docker/bashrc /etc/bash.bashrc
 RUN chmod a+rwx /etc/bash.bashrc
 
-RUN apt-get update && apt-get install -y unzip curl git software-properties-common
+RUN apt-get update && apt-get install -y unzip curl git software-properties-common libeigen3-dev
```

Comment:
For the last line of this file, it will fail as Go is not installed.
We can change it to 
`RUN make -f elasticdl/Makefile python_pb`
for now, and figure out how to install the PS Go version later.

Reply:
Done